actions/chats.js

```diff
@@ -139,7 +160,27 @@ export const fetchCreateChat = (name) => {
             .then((data) => {
                 dispatch(createChat(data));
             })
-            .catch((error) => console.log('Error:', error));
+            .catch((error) => {
+                console.log('Error:', error);
+                dispatch(setError(error));
```

Comment:
OK to remove console logs, you think? Setting the error in the redux store via the action SET_ERROR.